easybuild/framework/easyblock.py

```diff
@@ -1239,15 +1272,27 @@ def fetch_step(self, skip_checksums=False):
                 raise EasyBuildError("EasyBuild-version %s is newer than the currently running one. Aborting!",
                                      easybuild_version)
 
+        if build_option('extended_dry_run'):
+
+            print_msg("Available download URLs for missing sources/patches:", silent=self.silent, prefix=False)
```

Comment:
@verdurin: I see your point. I'll drop the 'required' part, and just make it "Available download URLs for sources/patches:"